### DISCUSSION TITLE
Modifying the default Fluent style for TrayIcon?

### DISCUSSION THREAD

**Discussion by Mtax-Development** [AvaloniaUI/Avalonia#8528] [Q&A]
I added the following style to an element inheriting from the `Application` class:

```
FluentTheme theme = new FluentTheme(new Uri("avares://" + Program.name)) {Mode = FluentThemeMode.Light};
Styles.Add(theme);
```

Is there a way to modify this style? My use for this case is modifying the menu of `TrayIcon` , as a style appears to be required to create its menu. I'd like to change the highlight color of the menu that is set when a menu option is hovered/clicked by the cursor. I'd assume this would have an effect only on Windows, as it is handled by the OS in other cases, but that is fine.

**Comment by maxkatz6** [comment]
Yes, if native menu is available, it will be used.
Otherwise, simple popup with templated menu is shown.
In current release, we don't have native menu implementation for windows.

Answering your question, menu shown in the TrayIcon is the same MenuFlyoutPresenter as it is normally shows in Avalonia menus. If you style MenuItem globally, it will be applied on the tray icon menu as well.
